man/machinery-list.1.md

```diff
@@ -4,13 +4,15 @@
 ### SYNOPSIS
 
 `machinery list`
+    NAME
```

Comment:
I just looked that up in the description for our remove command. What would be the right approach here [NAME , NAME] ?